yew/src/agent/worker/private.rs

```diff
@@ -87,6 +109,23 @@ where
     _agent: PhantomData<AGN>,
 }
 
+impl<AGN> PrivateBridge<AGN>
+where
+    AGN: Agent,
+    <AGN as Agent>::Input: Serialize + for<'de> Deserialize<'de>,
+    <AGN as Agent>::Output: Serialize + for<'de> Deserialize<'de>,
+{
+    /// Send a message to the worker, queuing it up if necessary
```

Comment:
I think it's better to mention exactly what's queued (I assume it's the message but it could be the worker, you never know 😁). This also has a minor style "improvement".
I always worry when I make these suggestions that they might come accross as condescending (please let me know if they do – if I can make them friendlier I'd appreciate suggestions as to how) 🤷🏼.
```suggestion
    /// Send a message to the worker, queuing the message if necessary
```